Test/civl/inductive-sequentialization/Gauss.bpl

```diff
@@ -0,0 +1,58 @@
+// RUN: %boogie "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+var {:layer 0,2} x:int;
+
+type {:pending_async}{:datatype} PA;
+function {:constructor} ADD(i: int) : PA;
+
+////////////////////////////////////////////////////////////////////////////////
+
+function trigger(i: int) : bool { true }
+
+procedure {:atomic}{:layer 1}
+{:IS "MAIN'","INV"}{:elim "ADD"}
+SUM (n: int)
+returns ({:pending_async "ADD"} PAs:[PA]int)
+modifies x;
+{
+  assert n >= 0;
+  assert trigger(0); // base hint
+  PAs := (lambda pa: PA :: if is#ADD(pa) && 1 <= i#ADD(pa) && i#ADD(pa) <= n then 1 else 0);
```

Comment:
Because PA has only one constructor, is#Add(pa) is always true.

Reply:
Yes, but I decided to put it nevertheless, in case somebody is playing with the example and adds another constructor.